ska_conda/ska_builder.py

```diff
@@ -11,32 +11,54 @@ class SkaBuilder(object):
 
     def __init__(self, ska_root=None):
```

Comment:
Add `user='sot', git_repo_path='git@github.com:{user}/{name}.git'` kwargs (and set corresponding instance attrs).